conf.py

```diff
@@ -90,7 +90,8 @@
     "github_user": "conan-io", # Username
     "github_repo": "docs", # Repo name
     "github_version": "develop2", # Version
-    "conf_py_path": "/" # Path in the checkout to the docs root
+    "conf_py_path": "/", # Path in the checkout to the docs root
+    "enable_cookies": os.getenv("ENABLE_COOKIES_BANNER", True)  # enable cookies banner
```

Comment:
Shouldn't this default not be False? That way local users won't ever get this and they dont have to look inito the code to find out how to disable them, and in the CI we can always set thiis to True

Reply:
If we guarantee that we don't forget it in CI, because it would be problematic if CI fails to set this and the docs go live without the cookies notice

Reply:
I would also vote for leaving the default to True, I'm afraid something fails setting the variable and we deploy without the banner.